API/Controllers/CustomerController.cs

```diff
@@ -62,11 +61,11 @@ public IActionResult Create(Customer customer)
         }
 
         [HttpDelete]
-        public IActionResult Delete(long id)
+        public virtual IActionResult Delete(long id)
         {
```

Comment:
[Comentário Acima](https://github.com/LuanFortunatoW/EverestBackendDotnet/pull/2/files#r1027898541)
```suggestion
        public IActionResult Delete(long id)
        {
```